pytest_describe/plugin.py

```diff
@@ -78,10 +89,6 @@ def collect(self):
     def _getobj(self):
         return self._importtestmodule()
 
-    def _makeid(self):
```

Comment:
By passing nodeid directly, we don't need this. Also, newer versions of pytest no longer use `_makeid`.

Reply:
Right, this was removed [here](https://github.com/pytest-dev/pytest/pull/3291) in pytest 3.5, and we now require pytest 4.0.